### ISSUE TITLE
Recent messaged contact are not accessible to the screenreaders

### ISSUE THREAD

**Issue by Harreth** [sindresorhus/caprine#1256]
A blind user of the app, reporting that the recent chats contact names are not getting labeled for accessibility nor presented by the app to the screenreader 
It reads the list of recent  messages like this when I errow down through it:
link

link

link

link
------
where it says "link" it should announce the messaged contact name as usual as it is with all web browsers. steps to reproduce it. -->
Use any screenreader E.G a trial of JAWS for windows or narrator
and arrow throgh the list

---

Caprine 2.43.0
Electron 7.1.10
win32 10.0.18363
Locale: en-US

**Comment by dusansimic** [comment]
Hi @Harreth. Sorry for waiting such a long time for the reply. Sadly, Caprine can't help here. We implement several additional features to the app but the accessibility of the website itself ([messenger.com](https://messenger.com)) relies on Facebook. They develop the website.

You could report the problem from the app (website) directly to Facebook. Click on your profile picture (bottom right corner), and in the menu that shows up, click "Report a problem". I'm not sure if that part of the app is accessible tho.

If you need more help, let me know.
